src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure [] figures = new Figure[6];
```

Comment:
All magic numbers in your code should be constants. The number of figures (6 in this case) should be extracted as a constant to improve code readability and maintainability. For example, private static final int FIGURE_COUNT = 6; Then you can use this constant instead of the magic number.